clients/include/testing_matmul.hpp

```diff
@@ -631,6 +662,37 @@ void testing_matmul(const Arguments& arg)
         CHECK_DEVICE_ALLOCATION(dBias[i]->memcheck());
         CHECK_DEVICE_ALLOCATION(dScaleAlphaVec[i]->memcheck());
         CHECK_DEVICE_ALLOCATION(dBias_C[i]->memcheck());
+        if(arg.use_e)
+        {
+            dE[i] = new device_vector<To>(size_E[i], 1, HMM);
+            CHECK_DEVICE_ALLOCATION(dE[i]->memcheck());
+        }
+        else
+        {
+            dE[i] = nullptr;
+        }
+
+        if(block_count > 1)
```

Comment:
is that possible to allocate block_count x size_X in one shot?
That separating dX[i] and alt_dX[i] produces many duplicated codes.

Reply:
Maybe next version with other changes?